src/lib/pipeline.ml

```diff
@@ -324,9 +328,11 @@ module Construct = struct
        make_target }
       bam_pair
 
-end
+  let rna_hla_typer fastq_sample = Rna_hla_typer fastq_sample
```

Comment:
This should also be `seq2HLA` for users of `Pipepline.Construct` to have future-proof and readable pipelines.